docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -32,7 +32,9 @@ For more information about KPA and HPA, see the documentation on [Supported Auto
         metadata:
           annotations:
             autoscaling.knative.dev/metric: "concurrency"
+            autoscaling.knative.dev/target-utilization-percentage: "70" 
     ```
+# autoscaling.knative.dev/target-utilization-percentage annotation for "Concurrency" sepcifies a percentage value which when reached autoscaling is performed
```

Comment:
This breaks the page formatting -- these need to be indented and have the `#` removed

<img width="451" alt="image" src="https://github.com/knative/docs/assets/44698588/6a0f9f9c-a190-47ce-a415-00b6be877578">

Reply:
A preview gets built on each commit you add: https://github.com/knative/docs/pull/5629#issuecomment-1637900462 (or you can build locally and preview that way too)

Reply:
Thanks @psschwei  for sharing information about preview feature. Currently I am redirected to knative homepage when I click the preview link. https://deploy-preview-5629--knative.netlify.app/

Reply:
Hello @psschwei  Thanks for showing the link. I have submitted another PR. Kindly review. Thanks!